pkg/backup/backup/backup_manager.go

```diff
@@ -85,7 +81,21 @@ func (bm *backupManager) syncBackupJob(backup *v1alpha1.Backup) error {
 	name := backup.GetName()
 	backupJobName := backup.GetBackupJobName()
 
-	_, err := bm.jobLister.Jobs(ns).Get(backupJobName)
+	err := backuputil.ValidateBackup(backup)
+	if err != nil {
+		uErr := bm.statusUpdater.Update(backup, &v1alpha1.BackupCondition{
+			Type:    v1alpha1.BackupInvalid,
+			Status:  corev1.ConditionTrue,
+			Reason:  "InvalidSpec",
+			Message: err.Error(),
+		})
+		if uErr != nil {
+			glog.Warningf("Update condition %s failed for backup %s/%s", v1alpha1.BackupInvalid, ns, name)
```

Comment:
If the update failed, the error log will be output in the update method, so I think it is not necessary to repeatedly output the error log.

Reply:
updated